plugin/acl/README.md

```diff
@@ -8,6 +8,9 @@
 
 With `acl` enabled, users are able to block or filter suspicious DNS queries by configuring IP filter rule sets, i.e. allowing authorized queries to recurse or blocking unauthorized queries.
```

Comment:
```suggestion
With `acl` enabled, users are able to block or filter suspicious DNS queries by configuring IP filter rule sets, i.e. allowing authorized queries or blocking unauthorized queries.
```

In addition to the source IP confusion, I think users in  #4697 may have misinterpreted "recurse" here, and thought this meant it would block forwarding to upstream recursive servers (but not to local authoritative zones).  What is actually meant here is that it either allows or blocks the query from passing to the rest of the plugin chain (but this is not "recursion" in a DNS sense).  Removing "to recurse" makes this clearer and more direct.